### ISSUE TITLE
What is the difference between Start and i.Start

### ISSUE THREAD

**Issue by FriederikeHanssen** [PoisonAlien/maftools#968]
**Describe the issue**

Hi! I have strelka called, vep annotated VCF files that I converted with vcf2maf. I am inspecting the maf file and saw that there are two start position columns: `Start` and `i.Start` and they don't have the same values, i.e.

```
Start:

my_maf@data[["Start_Position"]]
   [1]  27255190  29496908  15937550  86564268
   [5] 201984336  70442592 108178623 108199747

my_maf@data[["i.Start_Position"]]
   [1]  27255386  29496924  15938170  86564580
   [5] 201984398  70442614 108178691 108199887

```

I was wondering why that is and what each column means. 

**Command**
Please post your commands and the output (errors or any unexpected output)

```r
```

**Session info**
Run `sessionInfo()` and post the output below

```r

attached base packages:
[1] stats4    grid      stats     graphics 
[5] grDevices utils     datasets  methods  
[9] base     

other attached packages:
 [1] tidyr_1.3.0           GenomicRanges_1.52.0 
 [3] GenomeInfoDb_1.36.1   IRanges_2.34.1       
 [5] S4Vectors_0.38.1      BiocGenerics_0.46.0  
 [7] ggpubr_0.6.0          ggplot2_3.4.3        
 [9] gridExtra_2.3         maftools_2.16.0      
[11] dplyr_1.1.2           ComplexHeatmap_2.16.0
[13] kableExtra_1.3.4      knitr_1.43           
[15] cowplot_1.1.1         forcats_1.0.0        
[17] patchwork_1.1.3      

loaded via a namespace (and not attached):
 [1] tidyselect_1.2.0        viridisLite_0.4.2      
 [3] R.utils_2.12.2          bitops_1.0-7           
 [5] fastmap_1.1.1           RCurl_1.98-1.12        
 [7] digest_0.6.33           lifecycle_1.0.3        
 [9] cluster_2.1.4           survival_3.5-7         
[11] magrittr_2.0.3          compiler_4.3.1         
[13] sass_0.4.7              rlang_1.1.1            
[15] tools_4.3.1             utf8_1.2.3             
[17] yaml_2.3.7              data.table_1.14.8      
[19] ggsignif_0.6.4          xml2_1.3.5             
[21] RColorBrewer_1.1-3      abind_1.4-5            
[23] withr_2.5.0             purrr_1.0.2            
[25] R.oo_1.25.0             fansi_1.0.4            
[27] colorspace_2.1-0        scales_1.2.1           
[29] iterators_1.0.14        cli_3.6.1              
[31] rmarkdown_2.24          crayon_1.5.2           
[33] generics_0.1.3          rstudioapi_0.15.0      
[35] httr_1.4.7              rjson_0.2.21           
[37] cachem_1.0.8            DNAcopy_1.74.1         
[39] stringr_1.5.0           zlibbioc_1.46.0        
[41] splines_4.3.1           rvest_1.0.3            
[43] parallel_4.3.1          XVector_0.40.0         
[45] matrixStats_1.0.0       vctrs_0.6.3            
[47] webshot_0.5.5           Matrix_1.6-1           
[49] jsonlite_1.8.7          carData_3.0-5          
[51] car_3.1-2               GetoptLong_1.0.5       
[53] rstatix_0.7.2           clue_0.3-64            
[55] systemfonts_1.0.4       foreach_1.5.2          
[57] jquerylib_0.1.4         glue_1.6.2             
[59] codetools_0.2-19        stringi_1.7.12         
[61] shape_1.4.6             gtable_0.3.3           
[63] munsell_0.5.0           tibble_3.2.1           
[65] pillar_1.9.0            htmltools_0.5.6        
[67] GenomeInfoDbData_1.2.10 circlize_0.4.15        
[69] R6_2.5.1                doParallel_1.0.17      
[71] evaluate_0.21           lattice_0.21-8         
[73] highr_0.10              R.methodsS3_1.8.2      
[75] png_0.1-8               backports_1.4.1        
[77] broom_1.0.5             bslib_0.5.1            
[79] svglite_2.1.1           xfun_0.40              
[81] pkgconfig_2.0.3         GlobalOptions_0.1.2    
```

**Comment by FriederikeHanssen** [comment]
The above was generated whn simulatenously reading in data for all patients in my cohort (~100). When i only read in on epatient, it looks like the i.Start positions are used as "Start".

**Comment by PoisonAlien** [comment]
Hi,
Does any your MAF files have by any chance contain duplicated `Start_Position` columns?

**Comment by FriederikeHanssen** [comment]
Thanks for the super fast reply and the hint. 

I found a completely duplicated line in the maf file, the original vcfs don't have the position duplicate. Something must be going wrong when I convert. I'll take a look. 

How do the two start position relate to this?

**Comment by PoisonAlien** [comment]
I am not really sure if this is really the problem. My guess is that when you have duplicated column names, `data.table::rbindlist()` - which is used for concatenating two (or more) `data.tables` - tries to match the columns before doing so. This might cause conflict when there are more than one column with the same name. 
Its just a guess.. If you could share a reproducible example, I can have a proper look.

**Comment by FriederikeHanssen** [comment]
For anyone stumbling over this thread later: apparently using `bcftools view -R` occasionally duplicates entries, while as with -T I get unique ones only. 

I also ended up having duplicate entries when subsetting my maf object with GRanges to only look at specific regions. It might very well be me misusing something there.  I have resorted to preprocessing it with bcftools instead using `-T`. 

I'll try to generate an example with dummy data, I can't share my vcf files. Maybe for now a warning could be great. On the other hand, my mistake for not cleaning the data properly. 😬 🙈 

In any case thank you for the quick help 🙏
